src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -76,6 +76,13 @@
 
     public static final VoltageInitModeOverride VOLTAGE_INIT_MODE_OVERRIDE_DEFAULT_VALUE = VoltageInitModeOverride.NONE;
 
+    public enum TransformerVoltageControlMode {
```

Comment:
More an algo than a mode?

Reply:
I have chosen `Mode` for coherence with the rest of the parameters.